tests/components/input-for-test.js

```diff
@@ -228,7 +228,6 @@ test('binds label to input field across re-rendering', function(assert) {
   });
   var reRenderedInput = this.$().find('input');
   var reRenderedLabel = this.$().find('label');
-  assert.notEqual(initialInput.prop('id'), reRenderedInput.prop('id'));
```

Comment:
Previously the component was being recreated on every change, that's why the id was changing. Now the id does not change.